gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -2471,6 +2471,9 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
          print *, '       Nbits: ',       list(n)%nbits
          print *, '      Slices: ',       list(n)%Slices
          print *, '     Deflate: ',       list(n)%deflate
+         if (associated(list(n)%chunksize)) then
+            print *, '   ChunkSize: ',       list(n)%chunksize
```

Comment:
Bonus points if you convert this section to use pflogger instead ...

Reply:
That might be a bigger PR. 😄

Reply:
True, but technically as-is, this is a step in the slightly wrong direction.

Reply:
I've created an issue about it (#1306). I can see how to do most of it pretty easily. Just a couple places where we do `ADVANCE='NO'` I might need your help with